server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -202,7 +202,10 @@ public enum MetadataVersion {
     IBP_3_7_IV4(19, "3.7", "IV4", false),
 
     // Add ELR related supports (KIP-966).
-    IBP_3_8_IV0(20, "3.8", "IV0", true);
+    IBP_3_8_IV0(20, "3.8", "IV0", true),
+
+    // Introduce version 1 of the GroupVersion feature (KIP-848).
+    IBP_4_0_IVO(21, "4.0", "IV0", false);
```

Comment:
@dajac : It seems there is a typo. IBP_4_0_IVO should be IBP_4_0_IV0. The last zero was mis-typed as an O.

Reply:
Oh, good catch. Let me fix this.

Reply:
https://github.com/apache/kafka/pull/16181 @junrao Could you please review it?